module/organisations/helper.js

```diff
@@ -852,15 +878,17 @@ module.exports = class OrganisationsHelper {
    * check the user has permission for Org odmin or user admin
    * @method
    * @name _checkUserAdminAccess
-    * @returns {json} Response consists of profile data and user permission as boolean.
+   * @param {String} userId -  user id
+   * @param  {String} token  - user access token
+   * @param  {String} organisationId  - organisation id 
+   * @returns {json} Response consists of profile data and user permission as boolean.
 */
-
 function _checkUserAdminAccess(token, userId, organisationId) {
```

Comment:
@rakeshSgr - Function should take in userId, orgId and return array of all possible roles.